includes/class-amp-theme-support.php

```diff
@@ -100,8 +100,13 @@ public static function init() {
 			return;
 		}
 
+		// @todo Rename to query var to indicate whether sources should be obtained. It's sources that are needing to be to be conditional.
+		AMP_Validation_Manager::init( array(
+			'locate_sources' => AMP_Validation_Manager::should_validate_response(),
+			'debug'          => isset( $_REQUEST[ AMP_Validation_Manager::DEBUG_QUERY_VAR ] ), // phpcs:ignore WordPress.CSRF.NonceVerification.NoNonceVerification
```

Comment:
No cap check at this specific processing time on purpose?

Reply:
The `debug` flag is primarily used as a flag for whether to `remove_source_comments`. This only runs if `AMP_Validation_Manager::should_validate_response()` returns `true`, and that method does the cap check. It's used in one more place as a way to prevent validation errors from being sanitized, and here no cap check is involved, which may or may not be a good idea.

But I think this debug flag should go away entirely in favor of giving the ability to toggle whether individual validation errors should be sanitized or not in a preview interface. There should then be a separate param specifically for skipping `remove_source_comments`.